plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergSplit.java

```diff
@@ -19,6 +19,7 @@
 import com.google.common.collect.ImmutableMap;
 import io.trino.plugin.iceberg.delete.TrinoDeleteFile;
 import io.trino.spi.HostAddress;
+import io.trino.spi.SplitWeight;
```

Comment:
in commit message

> The weight is equal to the file size divided by the target split size.

 "equal to the file size" or "equal to portion of the file in bytes"? split can read a file, or a section of a file

Why divided? 
it looks like we're normalizing numbers to `(0,1)`, but i can imagine some splits to be above "target split size"
also, dividing doesn't seem to buy us anything, so let's use absolute numbers (number of bytes)